imports/ui/components/CosmosErrors.jsx

```diff
@@ -84,7 +84,7 @@ export default class CosmosErrors extends Component {
                         }
                         
                         this.state = {
-                            error: (error.codespace && error.code && errors[error.codespace][error.code]) || errors.sdk[1],
+                            error: (error.codespace && error.code && errors[error.codespace] && errors[error.codespace][error.code] && errors[error.codespace][error.code]) || "Unknown error",
```

Comment:
Only one ` && errors[error.codespace][error.code]` is needed. I was trying to tell you where to add the code.